src/mkdocs_autorefs/references.py

```diff
@@ -10,7 +10,10 @@
 from markdown.inlinepatterns import REFERENCE_RE, ReferenceInlineProcessor
 from markdown.util import INLINE_PLACEHOLDER_RE
 
-AUTO_REF_RE = re.compile(r'<span data-mkdocstrings-identifier=("?)(?P<identifier>[^"<>]*)\1>(?P<title>.*?)</span>')
+AUTO_REF_RE = re.compile(
+    r"<span data-(?P<kind>autorefs-identifier|autorefs-optional|mkdocstrings-identifier)="
```

Comment:
I guess `mkdocstrings-identifier` is just for backward compatibility?

Reply:
Yes. Just implicitly deprecated.